packages/esm-patient-forms-app/src/forms/forms-see-all.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useEffect } from 'react';
+import dayjs from 'dayjs';
+import first from 'lodash-es/first';
+import ContentSwitcher from 'carbon-components-react/es/components/ContentSwitcher';
+import Switch from 'carbon-components-react/es/components/Switch';
+import FormView from './form-view-see-all.component';
+import styles from './forms.component.scss';
+import { EmptyState } from '@openmrs/esm-patient-common-lib';
+import { useTranslation } from 'react-i18next';
+import { toFormObject } from './forms.resource';
+import { filterAvailableAndCompletedForms } from './forms-utils';
+import { Form } from '../types';
+import EmptyFormView from './empty-form.component';
+import { getObjectFHIR } from './get-FHIR-object';
+import { openmrsFetch } from '@openmrs/esm-framework';
+import _ from 'lodash';
```

Comment:
Please use `lodash-es` and a direct submodule import.

You see that already in the same module, e.g.:

```js
import first from 'lodash-es/first';
```

Reply:
The corrections have been made.